2024/political-donations-by-cycle/main.js

```diff
@@ -0,0 +1,487 @@
+    },
+    title: {
+        text: 'Cumulative Donations by Cycle from 2016-2022',
+        align: 'center'
+    },
+    subtitle: {
+        text: 'The 2020 Election saw record breaking donations by UChicago professors and nationwide.',
+        align: 'center',
+        style: {
+            fontSize: '15px'
+        }
+    },
+    xAxis: {
+        type: 'datetime',
+        labels: {
+            format: '{value:%b %Y}'
+        },
+        tickInterval: 30 * 24 * 3600 * 1000 * 4
+    },
+    yAxis: {
+        title: {
+            text: ''
+        },
+        labels: {
+            enabled: false
+        },
+        tickPositions: yAxisLabels,
+        endOnTick: false
+        //maxPadding: .5
+    },
+    plotOptions: {
+        spline: {
+            lineWidth: 7,
+            states: {
+                hover: {
+                    lineWidth: 8
+                }
+            },
+            marker: {
+                enabled: false
+            },
+            label: {
+                enabled: false
+            }
+        }
+    },
+    legend: {
+        enabled: false
+    },
+    tooltip: {
+        valuePrefix: '$',
+        xDateFormat: '%b %Y'
+    },
+    annotations: [
+        {
+            draggable: '',
+            labelOptions: {
+                borderRadius: 0,
+                backgroundColor: 'rgba(255, 255, 255, 0)',
+                borderWidth: 0,
+                borderColor: '#AAA'
+            },
+            labels: [
+                {
+                    point: {
+                        xAxis: 0,
+                        yAxis: 0,
+                        x: Date.UTC(2016, 11, 1),
+                        y: 350000
+                    },
+                    style: {
+                        fontSize: '14px'
+                    },
+                    text: '2016 Presidential Election<br><div style="color:#C16622"><b>$295,038<b>'
+                },
+                {
+                    point: {
+                        xAxis: 0,
+                        yAxis: 0,
+                        x: Date.UTC(2018, 9, 1),
+                        y: 350000
+                    },
+                    style: {
+                        fontSize: '14px'
+                    },
+                    text: '2018 Midterm Election<br><div style="color:#800000"><b>$384,179<b>'
+                },
+                {
+                    point: {
+                        xAxis: 0,
+                        yAxis: 0,
+                        x: Date.UTC(2020, 3, 1),
+                        y: 800000
+                    },
+                    style: {
+                        fontSize: '14px'
+                    },
+                    text: '2020 Presidential Election<br><div style="color:#155F83"><b>$1,289,469<b>'
+                },
+                {
+                    point: {
+                        xAxis: 0,
+                        yAxis: 0,
+                        x: Date.UTC(2022, 8, 1),
```

Comment:
`new Date(2022, 8, 1)` would be a better move, that way we don't have any problems with potential timezone issues.

Reply:
ok just a note for the future, highcharts needs a timestamp in milisecond to have to do `new Date(2022, 8, 1).getTime()`

Reply:
Perf, that works for me!